public/lessons/react/routing_in_react.md

```diff
@@ -192,6 +192,15 @@ const rootElement = document.getElementById("root");
 ReactDOM.render(<Router />, rootElement);
 ```
 
+# Lab: React Nav Bar
```

Comment:
This is great, React router is pretty confusing to use at first, maybe this lab will help clear up some of that confusion